lib/styles/react-joyride.scss

```diff
@@ -184,7 +184,7 @@ $joyride-tooltip-arrow-scale: ($joyride-tooltip-arrow-size / ($joyride-tooltip-a
 
     &.bottom-left,
     &.top-left {
-      .triangle {
+      .joyride-tooltip__triangle {
```

Comment:
Selector `joyride-tooltip__triangle` should be written in lowercase with hyphens